lib/modules/platform/pr-body.ts

```diff
@@ -9,6 +10,12 @@ export const prDebugDataRe = regEx(
   /\n?<!--renovate-debug:(?<payload>.*?)-->\n?/
 );
 
+const renovateConfigHashRe = regEx(
+  /\n?<!--renovate-config-hash:(?<payload>.*?)-->\n?/
+);
+
+const prCheckboxRe = regEx(/- (?<checkbox>\[[\sx]]) <!-- rebase-check -->/);
+
```

Comment:
https://github.com/renovatebot/renovate/pull/17633#discussion_r964369588
commet:
>i don't think we need the extra capture group. what's the benefit? i think a test instead of exec will be faster, as we don't need any position information.
>
> why it's exported?
>
>do we really use the unchecked match?

reply:
> https://github.com/renovatebot/renovate/pull/17633#discussion_r964494404